test/fixtures/output/schema-registry/region/list-all.golden

```diff
@@ -1,4 +1,4 @@
-    ID    | Cloud | Region Name |    Display Name    |   Packages    
-----------+-------+-------------+--------------------+---------------
-  sgreg-1 | aws   | us-east-2   | Ohio (us-east-2)   | [advanced]    
-  sgreg-2 | gcp   | us-central1 | Iowa (us-central1) | [essentials]  
+    ID    |        Name        | Cloud | Region Name |  Packages   
+----------+--------------------+-------+-------------+-------------
+  sgreg-1 | Ohio (us-east-2)   | aws   | us-east-2   | advanced    
```

Comment:
I'm confused; isn't `us-east-2` the region ID? What is `sgreg-1`?

Reply:
Either way, `us-east-2` deserves a separate column. Customers will definitely want to parse it in a script.

Reply:
`sgreg-1` is the unique identifier the customer needs to provide while creating a schema registry cluster.

Ref - https://docs.confluent.io/cloud/current/api.html#tag/Clusters-(srcmv2)/operation/createSrcmV2Cluster
https://docs.confluent.io/cloud/current/api.html#tag/Regions-(srcmv2)/operation/getSrcmV2Region
https://docs.confluent.io/cloud/current/stream-governance/clusters-regions-api.html#read-a-region

Reply:
Oh interesting, so a user who wants to use `us-east-2` wouldn't type `us-east-2`, they'd type `sgreg-1`?

Reply:
Yes - 'sgreg-1' is the primary key (unique for each cloud provider and region)

Reply:
All of our other commands require users to pass a (cloud, region) pair, which also solves that issue (instead of the user needing to look up a Confluent-specific region ID), i.e. `confluent kafka cluster create --cloud --region`

Reply:
Any hope that we can do the same thing for Schema Registry instead of introducing Confluent-specific region IDs like `sgreg-1`?

Reply:
Yes, though I don't exactly know in great detail the reason for this decision. Afaik, since the region is an independent Confluent Resource/Object with dedicated set of APIs for it (https://docs.confluent.io/cloud/current/api.html#tag/Regions-(srcmv2)), as per the Confluent API Guideline, it needs to have an identifier.

Reply:
Well hopefully other APIs will be updated to match this style... things are just a bit inconsistent now.